kube/services/netpolicy/networkpolicy_fence_templ.yaml

```diff
@@ -23,6 +23,9 @@ spec:
       - podSelector:
           matchLabels:
              app: revproxy
+      - podSelector:
```

Comment:
Why does shiny need to communicate directly with the fence service?
Would it be better if client code accessed our services via our standard public endpoint, than have one way to do things for workspace clients like shiny and another for external clients ?